add diataxis homepage (#803)
charmed-kubernetes/kubernetes-docs: index.md

Google, Microsoft, and many other institutions run Kubernetes on Ubuntu because we focus on the latest container capabilities in modern kernels. That’s why it’s the top choice for enterprise Kubernetes, too.

[Find out more in the Charmed Kubernetes overview&nbsp;&rsaquo;](../overview)

<img src="https://assets.ubuntu.com/v1/843c77b6-juju-at-a-glace.svg" style="float:right; margin-left: 2rem; border: 0" alt="">

## In this documentation

<div class="md-table">
<table>
<thead>
<tr>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="/docs/tutorials">Tutorials</a><br>  Get started with this introduction to Charmed Kubernetes <br></td>
<td><a href="/docs/how-to">How-to guides</a> <br> Step-by-step guides covering key operations and common tasks</td>
</tr>
<tr>
<td><a href="/docs/explanation">Explanation</a> <br> Concepts - discussion and clarification of key topics</td>
<td><a href="/docs/reference">Reference</a> <br> Technical information - specifications, commands, architecture</td>
</tr>
</tbody>
</table>
</div>

---

## Project and community

> Snap and Snapcraft are members of the Ubuntu family. They're both open source projects that welcome community involvement, contributions, suggestions, fixes and constructive feedback.

Reviewer: Looks like the wrong project name here, see suggestion

```suggestion
Charmed Kubernetes is a member of the Ubuntu family. It's an open source project that welcomes community involvement, contributions, suggestions, fixes and constructive feedback.
```

* [Our Code of Conduct](https://ubuntu.com/community/code-of-conduct)

> * [Visit us on Slack at #canonical-kubernetes](https://communityinviter.com/apps/kubernetes/community)

Reviewer: The upstream Kubernetes docs at https://kubernetes.io/community/#discuss use a different URL for Slack invites. I would be tempted to use the same URL in ours. It ultimately redirects to the same place, at least for now, so it's not a huge deal and I would approve it as-is too.

```suggestion
* [Visit us on Slack at #canonical-kubernetes](https://slack.k8s.io/)
```
